src/app-layout/__tests__/utils.tsx

```diff
@@ -106,6 +108,27 @@ export function findActiveDrawerLandmark(wrapper: AppLayoutWrapper) {
   return drawer.find('aside');
 }
 
+export function isDrawerTriggerWithBadge(wrapper: AppLayoutWrapper, triggerId: string) {
```

Comment:
This probably should be a public test-util, but this would require releasing drawers publicly first. Shipping it as an internal utility for now